code/mail/SESMailer.php

```diff
@@ -18,6 +18,13 @@ class SESMailer extends \Mailer {
 	 * @var SesClient
 	 */
 	private $client;
+
+	/**
+     * Allows QueuedJobs module to be bypassed when sending emails
+     *
+     * @var boolean
+     */
+    public $ignoreQueuedJobs = false;
```

Comment:
Can we reverse the naming on this?
From: `public $ignoreQueuedJobs = false;`
To:  `public $ignoreQueuedJobs = true;`